### ISSUE TITLE
Setting checkboxes for object boolean values has no focus

### ISSUE THREAD

**Issue by joaomoreno** [microsoft/vscode#146896]
The checkboxes for this new settings UI should have proper accessibility indicators such as focus (border) and active (background color change when mousedown) styles.

![Recording 2022-04-06 at 10 35 24](https://user-images.githubusercontent.com/22350/161932643-91bde7b0-cf38-4e74-915c-b8187b7de4b4.gif)

**Comment by rzhao271** [comment]
This issue seems to occur for the Workspace Trust checkbox too.
There is a focus indicator if one carefully uses keyboard nav to get the focus onto the checkbox, but there's no hover or active indicators.

**Comment by rzhao271** [comment]
I confirmed that boolean object settings such as "SCM: Diff Decorations Gutter Pattern" still don't have proper active/mousedown styling.
